build/android/cross

```diff
@@ -72,7 +72,7 @@ elif [ "$ARCH" = "armeabi-v7a" ]; then
     CXXFLAGS="$CFLAGS"
 elif [ "$ARCH" = "mips" ]; then
     # Note: mips is actually mipsel (i.e. little endian)
-    TOOLCHAIN_NAME=mipsel-linux-android
+    TOOLCHAIN_NAME=mips64el-linux-android
```

Comment:
This is an upgrade in the toolchain by which we always need to use `mips64el`. I have tested this by running the resulting example app for Android on a `mips` emulator with API 16.